src/internal/operators/catchError.ts

```diff
@@ -14,8 +14,18 @@ export function catchError<T, O extends ObservableInput<any>>(selector: (err: an
 /**
  * Catches errors on the observable to be handled by returning a new observable or throwing an error.
  *
+ * <span class="informal">It only listens to the error channel and ignores notifications.
+ * If an error is emitted it catches it and retrieves the error as parameter.
+ * It can either return a new `Observable` to swallow the error or rethrow an new error.</span>
```

Comment:
I think I want to move away from calling them "channels". I'm not sure people relate to that. Also, mapping to a new observable will not "swallow" the error, of you do `catchError(err => throwError(err))` 🤷‍♂

Simpler explanation:
```
Handles errors from the source observable, and maps them to a new observable. The error may also be rethrown, or a new error can be thrown to emit an error from the result.
```